client/odie/message/jump-to-recent.tsx

```diff
@@ -46,37 +46,13 @@ export const JumpToRecent = ( {
 		};
 	}, [] );
 
-	useEffect( () => {
-		const handleIntersection = ( entries: IntersectionObserverEntry[] ) => {
-			const entry = entries[ 0 ];
-			const visibility = entry.intersectionRatio; // percentage of visibility (0, 1)
-
-			setIsLastMessageVisible( visibility > 0 );
-		};
-
-		const options = {
-			root: lastMessageRef.current?.parentElement,
-			threshold: [ 0, 1 ],
-		};
-
-		const observer = new IntersectionObserver( handleIntersection, options );
-
-		if ( lastMessageRef.current ) {
-			observer.observe( lastMessageRef.current );
-		}
-
-		return () => {
-			observer.disconnect();
-		};
-	}, [ lastMessageRef ] );
-
-	if ( isMinimized && botSetting === 'supportDocs' ) {
+	if ( isMinimized && botNameSlug === 'wpcom-support-chat' ) {
```

Comment:
Perhaps we can move all instances of hardcoded `wpcom-support-chat` to use an enum?

Reply:
It's a type already, you cannot use anything different that 
![image](https://github.com/Automattic/wp-calypso/assets/5689927/49c8219b-2d6a-4a3d-bf76-f45fedc50a96)

Reply:
Yea, we may have to change this though if we ever have multiple allowed bots, but can worry about that later